particles/Tutorial/Kotlin/Demo/src/TTTGame.arcs

```diff
@@ -119,7 +119,7 @@ recipe GamePlayersDemoRecipe
     gameState: reads gameState
     myMove: writes playerTwoMove
     player: reads playerTwo
-  description `Kotlin Tutorial TicTacToe One Player Demo`
+  description `Kotlin Functional Tutorial TicTacToe One Player Demo`
```

Comment:
You could probably revert these changes if you are just going to replace the existing tutorial.